routes/web.php

```diff
@@ -510,4 +544,11 @@ function (): void {
     // User must be logged in to user currentuser routes
     Route::get('currentuser/manager', 'Api\ManagerApiController@showAuthenticated')
         ->middleware('auth');
+
+    // HR Portal - Claims
+    Route::resource('hr', 'HrAdvisorController');
```

Comment:
I think this needs to come out, it's trying to generate routes to methods in `App\Http\Controllers\HrAdvisorController.php`